pennylane/interfaces/execution.py

```diff
@@ -254,6 +254,7 @@ def inner_execute(tapes: Sequence[QuantumTape], **_) -> ResultBatch:
             tapes = tuple(qml.transforms.convert_to_numpy_parameters(t) for t in tapes)
         return cached_device_execution(tapes)
 
+    # TODO: set as pure_callback if interface is jax-jit

```

Comment:
How does this tie into the device change?

Reply:
tbh I don't remember and it doesn't seem to be a problem anymore. I bet I wrote this before realizing something about ParametrizedEvolution etc. and it actually doesn't apply. deleting this TODO